op-node/metrics/metrics.go

```diff
@@ -302,6 +302,7 @@ func NewMetrics(procName string) *Metrics {
 			// No label names here since peer ids would open a service attack vector.
 			// Each peer id would be a separate metric, flooding prometheus.
 			// See: https://prometheus.io/docs/practices/naming/#labels
```

Comment:
Wait does this go against the comment?

Reply:
Yes.

Reply:
I don't know.

Reply:
Why do we need to "actually track how peer scores are calculated on a per-peer basis". Is this something where we don't have insight into the system. Do we need this to make sure that the feature is working, or will we also need this on an ongoing basis to track the performance of the peer scores feature.

Reply:
This is the only metrics for peer scoring atm. So I don't see how else we would glean insight into peer scores.

Again, i'm not sure how to track this - I haven't touched prometheus metrics much so any direction here would be greatly appreciated.

Reply:
there should be an RPC api that exposes metrics on a peer to peer basis? i believe we can set up in grafana curl calls to build a chart

Reply:
> how else we would glean insight into peer scores

We just need to figure out metrics that don't include the peer ID. The limit with grafana is that if we include externally controllable data as a label, we expose ourselves to a DOS attack. Can do a distribution of scores. Can do events when we want to ban peers (& then use logs to figure out which peer we banned). Can do events when we raise/lower scores. I'm sure there are more that I haven't thought of.